src/components/avatar/Avatar.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import Icon, {TYPE as ICON_TYPE, COLOR as ICON_COLOR} from '../icons/Icon';
+import Icon, {TYPE as ICON_TYPE, COLOR as ICON_COLOR} from 'icons/Icon';
```

Comment:
@matikrk I saw @kdzwinel's PR and he moved DocsBlock and ContrastBox to `docs/components/`, not 'components'